.github/ISSUE_TEMPLATE/b-bug-report.yml

```
@@ -0,0 +1,87 @@
+name: Report a bug
+description: You encountered a bug in the geolocator plugin that you would like to report.
+title: "Bug: "
+labels: ["needs-triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to submit an issue!
+        To effectively tackle it, we require as much information as possible.
+  - type: checkboxes
+    attributes:
+      options:
+      - label: I have searched the [existing issues](https://github.com/baseflow/flutter-geolocator/issues)
+        required: true
+  - type: checkboxes
+    attributes:
+      label: Please select affected platforms
+      options:
+      - label: Android
+      - label: iOS
+  - type: textarea
+    attributes:
+      label: Steps to reproduce
+      description: Please tell us exactly how to reproduce the problem you are running into.
+      placeholder: |
+        1. ...
+        2. ...
+        3. ...
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Expected results
+      description: Please tell us what is expected to happen.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Actual results
+      description: Please tell us what is actually happening.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Code sample
+      description: |
+        Please create a minimal reproducible sample that shows the problem
+        and attach it below between the lines with the backticks.
+        
```

I'd really like to include the extra note asking issue reporters not to upload screenshots of text:

```suggestion
        Please create a minimal reproducible sample that shows the problem
        and attach it below between the lines with the backticks.
        
        Note: Please do not upload screenshots of text. Instead, use code blocks
        or the above mentioned ways to upload your code sample.
```